web/src/people.ts

```diff
@@ -766,6 +766,11 @@ export function sender_is_guest(message: Message): boolean {
     return false;
 }
 
+export function sender_is_deactivated(message: Message): boolean {
+    const sender_id = message.sender_id;
+    return sender_id ? get_non_active_human_ids().includes(sender_id) : false;
+}
```

Comment:
This is a very inefficient check -- you're spending O(n) time to just look up the user.

Reply:
Can't the caller just use `is_active_user_for_popover(message.sender_id)`?